mne/minimum_norm/inverse.py

```diff
@@ -1339,10 +1340,11 @@ def make_inverse_operator(info, forward, noise_cov, loose='auto', depth=0.8,
     """  # noqa: E501
     is_fixed_ori = is_fixed_orient(forward)
 
-    if fixed and loose not in [None, 'auto']:
-        warn('When invoking make_inverse_operator with fixed=True, the loose '
-             'parameter is ignored.')
-        loose = None
+    if fixed:
+        if loose not in [None, 'auto', 0.]:
+            warn('When invoking make_inverse_operator with fixed=True, the '
+                 'loose parameter is ignored.')
+        loose = 1. if loose is not None else loose  # keep deprecationwarning
```

Comment:
I don't get this. loose should not have any impact afterwards.
I would force loose=0. to be consistent with loose=0. -> fixed

Reply:
It does have an impact afterward (I originally had the code like what you suggest). It needs to stay `loose=1.` because we compute the depth prior (I think?) with free orientations then convert later.

Reply:
(If you think about how this code flowed before where `loose=None` means "free orientation" it makes a bit more sense)